.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Kindly add `test.md` in the `.gitignore` file and make sure it is ignored by git 😄